public/views/login/LoginController.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+(function () {
+
+    var app = angular.module('CS4570');
+
+    app.controller('LoginController', ['$http', '$scope', '$window', '$filter', '$location', '$rootScope', '$cookies',
+        function ($http, $scope, $window, $filter, $location, $rootScope, $cookies) {
+            var self = this;
+
+            if($cookies.get('email')) {
+                $location.path("/home");
+            }
+
+
+            self.user = {
+                email: '',
+                password: ''
+            };
+
+
+            self.enableSubmit = function () {
+                return self.user.email != '' && self.user.password != '';
```

Comment:
We need to do better validation here. Check if the email is a valid email and we should decide some criteria for the password also.